__tests__/foo.test.ts

```diff
@@ -6,7 +6,7 @@ test("foo", ()=>{
 });
 
 test("date", ()=>{
-    expect(date()).toEqual('2017/01/01/00');
+    expect(date()).toEqual('2017/01/01/01');
```

Comment:
nit: now that we have real tests set up, we can probably remove foo and foo.test.